.pre-commit-config.yaml

```diff
@@ -10,12 +10,13 @@ repos:
     hooks:
     -   id: ruff
         types_or: [
-            python, 
-            pyi, 
+            python,
+            pyi,
             # jupyter, # https://github.com/xdslproject/xdsl/issues/1273
             ]
         args: [ --fix, --exit-non-zero-on-fix ]
--   repo: https://github.com/psf/black
+-   repo: https://github.com/psf/black-pre-commit-mirror
```

Comment:
https://github.com/psf/black/blob/main/.pre-commit-hooks.yaml

```
# Note that we recommend using https://github.com/psf/black-pre-commit-mirror instead
# This will work about 2x as fast as using the hooks in this repository
```